### ISSUE TITLE
[Bug]Mozilla Nightly gets displayed instead of Mozilla Beta while using Remote debugging via USB

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#13108]
## Prerequisites
Enable "Remote debugging via USB" on Fenix

## Steps to reproduce
Connect the device Via USB
Open Firefox desktop and load in a new tab about:debugging
Connect with the Android device

### Expected behavior
Mozilla Beta should be displayed

### Actual behavior
Mozilla Nightly is displayed

### Device information

* Android device: all
* Fenix version: Fenix Beta 80.0.0-beta-1

### Notes
![image](https://user-images.githubusercontent.com/51314259/88892888-35cb0100-d24e-11ea-933a-9f4d04034b9c.png)
![image](https://user-images.githubusercontent.com/51314259/88892845-2481f480-d24e-11ea-8bde-f29488962c11.png)

**Comment by kbrosnan** [comment]
This sounds like an `about:debugging` bug, https://bugzilla.mozilla.org/enter_bug.cgi?product=DevTools&component=about%3Adebugging

**Comment by AndiAJ** [comment]
Filled https://bugzilla.mozilla.org/show_bug.cgi?id=1656170

**Comment by juliandescottes** [comment]
The hack we used to detect Fenix seems to no longer be valid.

We detected Fenix by checking that the package name contained "org.mozilla.fenix", but now it's been renamed to "org.mozilla.firefox" (makes sense I guess).
 
Note that this is also linked to https://github.com/mozilla-mobile/fenix/issues/2016 and https://bugzilla.mozilla.org/show_bug.cgi?id=1589805 . It would allow to remove the Fenix specific workarounds.
